examples_utils/benchmarks/run_benchmarks.py

```diff
@@ -553,3 +566,9 @@ def benchmarks_parser(parser: argparse.ArgumentParser):
         type=int,
         help="Maximum time allowed for any benchmark/variant (in seconds)",
     )
+
+    parser.add_argument(
```

Comment:
Nitpick: I like to define args alphabetically so theyre a little easier to find without a search

Reply:
I have changed this so it's at the start, other arguments aren't alphabetical though, so maybe you'll want to refactor that in the future

Reply:
Thanks! Yh being done in another PR I think